Assets/CharTest/Scripts/SeleneFire.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SeleneFire : MonoBehaviour
+{
+
+    [SerializeField]
+    private int primaryProjectiles;     //Number of projectiles in first wave of primary spell
+    [SerializeField]
+    private int primaryWaves;           //Number of waves in primary spell
+    [SerializeField]
+    private float primarySpeed;         //Speed of projectiles in primary spell
+    [SerializeField]
+    private int heavyProjectiles;       //Number of split heavy projectiles
+
+    public GameObject primaryPrefab;  
+    public GameObject heavyPrefab;
+
+    private Vector2 startShot;
+    [HideInInspector]
+    public Vector2 TarDir;
```

Comment:
Is there a reason you're making this public?

Reply:
It was in a previous iteration where I was testing HeavyBullet